src/theme/MDXComponents.js

```diff
@@ -124,10 +123,6 @@ function getCalicoVersionedComponent(version, componentName) {
       return require(`../../calico_versioned_docs/version-${version}/_includes/components/PodCidrSed`).default;
     case 'ReqsKernel':
       return require(`../../calico_versioned_docs/version-${version}/_includes/components/ReqsKernel`).default;
-    case 'ReqsSys':
-      return require(`../../calico_versioned_docs/version-${version}/_includes/components/ReqsSys`).default;
-    default:
```

Comment:
did you mean to remove the `default` case?

Reply:
Nope, thanks! Fixed.